src/components/UI/FullScreenButton/FullScreenButton.test.js

```diff
@@ -18,26 +18,36 @@ jest.mock('./FullScreenButton.messages', () => {
 });
 
 describe('FullScreenButton tests', () => {
+  beforeEach(() => {
+    document.exitFullscreen = jest.fn();
+    document.documentElement.requestFullscreen = jest.fn();
+  });
+
   test('default renderer', () => {
     shallowMatchSnapshot(<FullScreenButton onClick={() => {}} />);
   });
   test('on buttton click', () => {
```

Comment:
We can find a more descriptive name for this test. It can be something like `'requests fullscreen on click'`